keepalived/check/check_api.c

```diff
@@ -42,8 +42,9 @@
 #include "check_dns.h"
 
 /* Global vars */
-static checker_id_t ncheckers = 0;
+checker_id_t ncheckers = 0;
 list checkers_queue;
+list old_checkers_queue;
```

Comment:
I would prefer this to be a local variable in reload_check_thread() and be passed as a parameter to the necessary functions.